create_item.rb

```diff
@@ -0,0 +1,105 @@
+module CreateItems
+  #   Create person
+  def create_people
+    puts 'Do you want to create a student (1) or a teacher (2) ? [Input the number]:'
+    input = gets.chomp.to_i
+    case input
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else
+      puts 'Please Enter 1/2 please'
+      create_people
+    end
+  end
+
+  # create student
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+
+    parent_permission = ''
+
+    case gets.chomp.downcase
+    when 'y'
+      parent_permission = true
+
+    when 'n'
+      parent_permission = false
+    else
+      puts('please enter [Y/N]: ')
+    end
```

Comment:
- [ ] In the `create_student` method you are getting input from users and creating the student, This method does not follow `single responsibility principle`. Please make sure to separate these operations into different methods to achieve `single responsibility principle`.